tests/conftest.py

```diff
@@ -44,15 +38,19 @@
 
 NO_T2_REGIONS = ["eu-north-1", "ap-east-1", "me-south-1"]
 
+# TODO: refactor handling of versions, repo, image uris, validations for all frameworks
+TENSORFLOW_LATEST_VERSION = "2.2.0"
+TENSORFLOW_LATEST_1X_VERSION = "1.15.2"
```

Comment:
I think we can simplify the TF version stuff here and just use 2.x (like how the other frameworks just test one version for integ tests). I don't think the Python SDK behaves differently in our integ tests between TF versions 1.15.2 and 2.2.0

Reply:
sounds good, will simplify